src/HotChocolate/Core/src/Types/Types/Directives/OptInFeatureStabilityDirectiveExtensions.cs

```diff
@@ -0,0 +1,12 @@
+namespace HotChocolate.Types;
+
+public static class OptInFeatureStabilityDirectiveExtensions
+{
+    public static ISchemaTypeDescriptor OptInFeatureStability(
```

Comment:
We should have an extension method also for `IRequestExecutorBuilder` so that we can do schema wide things on the main builder.